governance/roles.md

```diff
@@ -0,0 +1,96 @@
+* Propose agenda items for meetings to ensure that open issues are discussed with the group when needed.
+* Serving as TAG leadership representative in CNCF project discussions and on community events.
+* Onboard and mentor new community members.
+* Mentor community members in a leadership role within the TAG.
+* Enforce and encourage diversity in the TAG efforts.
+* Enforce and encourage good communication in the TAG efforts following the [CNCF CoC](https://www.cncf.io/conduct/).
+* Support the TAG Chairs.
+
+Permissions:
+* Tech Leads have maintain access to the TAG GitHub repository.
+
+## Working Group Leads
+
+A working group chairs (usually 3) are experienced contributors to the TAG that applies to lead along one or two others a working group.
+Working groups as community governance structure are defined in the [CNCF TAG](https://github.com/cncf/toc/blob/master/tags) folder.
+TAG Chairs are responsible facilitating the working group applications and make sure that the process is done transparent, accessible and traceable for everyone.
+At least one of the TAG chairs or Tech Leads serves as the working group sponsor and provides guidance to the working group leads.
+Working group leads are signed off by the majority of the TAG Chairs and the TOC Liaisons.
+The [leadership election process is described here](template-leadership-election-process.md#working-group-lead).
+
+**A working group lead is expected to**:
+* Schedule, host, plan and facilitate meetings for the working group.
+* Provide technical direction unique to the working group.
+* Plan working group deliverables.
+* Report to the TAG on the working group status.
+* Evolve the working group.
+* Garner active participation.
+* Establish documents and correct permissions for contributions to occur.
+* Contribute content to the working group.
+* Onboard and setup the working group lead successor.
+
+Permissions:
+* WG Leads have write access to the TAG GitHub repository and CODEOWNERS entries for the working group folder.
+
+## TOC Liaison
+
+The [CNCF TAG](https://github.com/cncf/toc/blob/master/tags) process identifies a TOC Liaison.
+The TAG Chairs are responsible for establishing effective communication with the TOC Liaison, including further communication to the wider TOC upon request.
+
+The TOC Liaison will occasionally prioritize TAG activities, as needed by the TOC, to further the [CNCF mission](https://github.com/cncf/foundation/blob/master/charter.md#1-mission-of-the-cloud-native-computing-foundation).
+
+Permissions:
+* TOC Liaisons have admin access to the TAG GitHub repository.
+
+## Emeritus Chair
+
+After a TAG Chair completes their term, they transition into the role of Emeritus Chair.
+This role does not impose any obligations on the TAG, but it acknowledges their committed time and effort.
+The role also lets them share information about their previous work and contributions to the TAG.
+An Emeritus Chair retains the same rights/access as in their previous role.
```

Comment:
I lean to say emeritus chairs shouldn't retain admin permissions. There may be times when we need to remove access.

Reply:
What do you propose @joshgav? - WG Leads have 'write' access, TL's have 'maintain' and Chairs have 'admin'
Propose to drop to 'write' or 'maintain'?